bazel/repository_locations.bzl

```diff
@@ -162,12 +162,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Abseil",
         project_desc = "Open source collection of C++ libraries drawn from the most fundamental pieces of Google’s internal codebase",
         project_url = "https://abseil.io/",
-        version = "c8b33b0191a2db8364cacf94b267ea8a3f20ad83",
-        sha256 = "a7803eac00bf68eae1a84ee3b9fcf0c1173e8d9b89b2cee92c7b487ea65be2a9",
+        version = "20230802.0",
```

Comment:
Latest is `20230802.1` https://github.com/abseil/abseil-cpp/releases/tag/20230802.1